### DISCUSSION TITLE
Why my LinkFree Profile is Not Showing?

### DISCUSSION THREAD

**Discussion by AwesomeArunava** [EddieHubCommunity/LinkFree#3689] [Q&A] [answered]
Why my LinkFree Profile is Not Showing?

**Comment by kumarsonsoff3** [comment, marked as answer]
It's now live, here at: https://linkfree.eddiehub.io/AwesomeArunava

And this is why it was not showing!!
---
![image](https://user-images.githubusercontent.com/88786642/213845092-0f46ad93-1540-4847-892d-429306d859ee.png)

**Reply by eddiejaoude** [reply]
thank you @kumarsonsoff3 👍

**Reply by AwesomeArunava** [reply]
Thank you @kumarsonsoff3 ❤️

**Comment by webAbhi** [comment]
Hey @kumarsonsoff3, is this step compulsory to perform if i am only bothered about successfully running my local environment ?
i am unable to login to my account via localhost. BTW i am using docker to run it locally and created my profile online. am i missing anything here ?  this was the console error 👇👇.
![Screenshot 2023-07-23 at 2 02 22 PM](https://github.com/EddieHubCommunity/LinkFree/assets/25529176/7245e513-6f1c-436d-bc96-211a7f0a1a04)

**Reply by eddiejaoude** [reply]
You need to set the environment variables, you can learn more about them in our documentation https://linkfree.io/docs/environments/environment-variables

please note @webAbhi it is better to start a new discussion in the future, and not hijack another "answered" discussion

**Reply by webAbhi** [reply]
Thank you soo much for the reference and yes i will keep this thing in mind .
